test/tests/functional/pbs_reservations.py

```diff
@@ -2310,3 +2310,177 @@ def test_resv_reconfirm_holding_partial_nodes(self):
         solution = '(' + vn[0] + ':ncpus=2)+(' + resv_node2 + ':ncpus=1)'
         a = {'reserve_substate': '5', 'resv_nodes': solution}
         self.server.expect(RESV, a, id=rid)
+
+    @skipOnCpuSet
```

Comment:
Instead of skipping the test on cpuset machines try updating the test in such a way that it works on cpuset machine. (ex: instead of setting available ncpus to 2, take total num of ncpus available and submit the job accordingly)

Reply:
@Mahalty , as per our  offline discussion I left  this comment .
Since as we discussed if we take total num of ncpus available and submit the job accordingly then on those machine where ncpus=1 test will fail  , because  in first 2 steps we need 2 ncpus (1 for job and 1 for reservation)

we discuss few other scenarios as well, but try to summaries the reason as above :) . HTH